esp/src/src-react/components/SetLogicalFileAttr.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService, WsDali } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+class MyDaliService extends DaliService {
+    SetLogicalFileAttr(request: WsDali.SetLogicalFilePartAttrRequest): Promise<WsDali.ResultResponse> {
```

Comment:
Looks like "SetLogicalFileAttr" should be "SetLogicalFilePartAttr" instead (https://github.com/hpcc-systems/Visualization/blob/trunk/packages/comms/src/services/wsdl/WsDali/v1.04/WsDali.ts#L200). But also, I think these MyDaliService classes could be removed altogether, as the function that is being defined is already available in the extended object.